sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/InsetViewScreen.kt

```diff
@@ -0,0 +1,118 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.molecule.inset.InsetView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme.colors
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme.typography
+import uk.gov.hmrc.components.compose.ui.theme.hmrc_spacing_16
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.extension.addPlaceholderModifier
+
+@Composable
+fun InsetViewScreen() {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .verticalScroll(rememberScrollState())
+            .padding(hmrc_spacing_16)
+    ) {
+        Heading5(text = stringResource(id = R.string.heading_placeholder_plural),
+            modifier = Modifier.padding(bottom = hmrc_spacing_16))
+
+        InsetView(modifier = Modifier
+            .padding(bottom = hmrc_spacing_16)
+            .addPlaceholderModifier(),
+            childView = {
+                Text(
+                    text = stringResource(R.string.inset_placeholder_text),
+                    style = typography.body,
+                    textAlign = TextAlign.Start
+                )
+            }
+        )
+
+        Heading5(text = stringResource(id = R.string.heading_example_plural),
+            modifier = Modifier.padding(bottom = hmrc_spacing_16))
+
+        Card(
+            modifier = Modifier.fillMaxWidth(),
+            shape = RoundedCornerShape(0),
+            colors = CardDefaults.cardColors(containerColor = colors.hmrcWhiteBackground)) {
+            InsetView(modifier = Modifier.padding(hmrc_spacing_16),
+                childView = {
+                    Column() {
+                        Row {
+                            Text(
+                                text = stringResource(R.string.inset_example_text_line_1_description),
+                                modifier = Modifier.padding(end = 4.dp),
```

Comment:
Please can we use our defined dimens for spacing? eg. `hmrc_spacing_4` (not sure if we've defined them all yet, but we're already using `hmrc_spacing_16` in this file.. 
Same for the other spacing in this file. :)